src/Containers/VendorsList/Profile.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FeaturedProfile, IFeaturedProfileProps } from '../FeaturedProfile';
+import { HeaderRow, HeaderRowProps } from '../HeaderRow';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { flex, media } from '../../Utils/Mixins';
+
+export interface IProfileProps extends MainInterface, ResponsiveInterface,React.HTMLAttributes<HTMLDivElement> {
+    headerRowProps?: HeaderRowProps;
+    imageUrl?: string;
+    key: number;
+    name: string;
+    email: string;
+    profileProps?: IFeaturedProfileProps;
+}
+
+const MATCH_FIRST_LETTER = /\b\w/g;
+
+export const Profile: React.FC<IProfileProps> = ({
+    imageUrl, 
+    name, 
+    email,
+    key,
+    headerRowProps,
+    profileProps,
+    ...props
+}): React.ReactElement => {
+    /**
+     * Returns initials of a person's full name
+     * @param inputName{string} - person's name
+     */
+    const getInitials = (inputName: string) => {
+        const initials = inputName.match(MATCH_FIRST_LETTER) || [];
+        const profileInitials = ((initials.shift() || '') + (initials.pop() || '')).toUpperCase();
+        return profileInitials;
+    };
+
+    /**
+     * Returns profile component based on whether an imageurl is provided 
+     * @param inputName {string} - person's name
+     * @param image {string} - image's url
+     */
+    const getProfiles = (inputName: string, image?: string) => {
+        const profileInitials = getInitials(inputName);
+
+        switch(true) {
+        case image === undefined:
+            return (
+                <FeaturedProfile 
+                    {...profileProps}
+                    background='orange'
```

Comment:
is there any way to use theme here? background is a mandatory prop as a string

Reply:
const theme = useTheme();
Styled components hook.

Then theme.Colors.Etc